app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: post, local: true) do |form| %>
+  <% if post.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% post.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ]  put the if statement in a helper method that renders it's content instead fo having the if statement directly in the view since the content has more than one line you should use a partial to put the markup and render the partial within the helper method

Reply:
Done